### DISCUSSION TITLE
Can it work on a docker installation ?

### DISCUSSION THREAD

**Discussion by Taized** [hassio-addons/addon-spotify-connect#161] [Q&A] [answered]
Hi,

I'm a beginner with HA, 
I would like to use the Spotify connect on a docker installation of HA, but I still don't understand if it is possible or not…

Thank you in advance,
Regards :)

**Comment by frenck** [comment, marked as answer]
Add-ons are not available on Home Assistant Container installation types.

If you want to run something like this, you'll need to check e.g., DockerHub or similar for solutions that fit your set up of choice.

**Reply by Taized** [reply]
Ok I'm gonna check that !
Thank you a lot for your very fast answer :)
